tests/jobs/stea/test_stea.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import absolute_import
+import pytest
+from semeio.jobs.scripts.fm_stea import main_entry_point
+import importlib
+
+import sys
+
+if sys.version_info >= (3, 3):
+    from unittest.mock import patch
+else:
+    from mock import patch
+
+from stea import SteaResult, SteaKeys, SteaInput
+import os
+
+TEST_STEA_PATH, _ = os.path.split(os.path.abspath(__file__))
+
+
+def test_import_stea():
+    assert importlib.import_module("stea")
```

Comment:
I might be a bit confused, but what is the purpose of this test?

Reply:
testing that stea is installed (legacy of previous testing) ... but might also go away, since stea is now part of setup.py dependencies installation.

Reply:
removed